### PR TITLE
[buildifier] Warn and fix loads from the same file.

### PR DESCRIPTION
**Pull request by ttsugriy**
Loads like
```
load(":f.bzl", "s1")
load(":f.bzl", "s2")
```
can be simplified as
```
load(":f.bzl", "s1", "s2")
```
which is more concise and less error prone.

### CODE REVIEW

File: warn/warn.go [hunk 1]
```diff
@@ -663,6 +693,7 @@ var FileWarningMap = map[string]func(f *build.File, fix bool) []*Finding{
 	"http-archive":       nativeHTTPArchiveWarning,
 	"integer-division":   integerDivisionWarning,
 	"load":               unusedLoadWarning,
+	"same-origin-load":   sameOriginLoadWarning,
```
Comment (pmbethe09) [inline warn/warn.go]: preserve alphabetical here -- so push down to the natural spot

**Comment by vladmos** [comment]
Please also add the warning description to the WARNINGS.md file.

**Comment by ttsugriy** [comment]
> Please also add the warning description to the WARNINGS.md file.

@vladmos, I actually had a warning description initially, but I removed it because I somehow got under impression that those were for buildozer fix instead of buildifier, since there is only one reference to buildifier and a lot of reference including command examples for buildozer :) Now that you've removed the confusion for me, I readded the warning description.

File: WARNINGS.md [hunk 2]
```diff
@@ -141,6 +141,34 @@ or at the beginning of a rule.
 
 --------------------------------------------------------------------------------
 
+## <a name="load"></a>Same label is used for multiple loads
```
Comment (vladmos) [inline WARNINGS.md]: Please rename to `same_origin_load`

Comment (ttsugriy) [reply WARNINGS.md]: sorry about that and thanks for the catch!

**Comment by vladmos** [comment]
Thanks!

**Comment by ttsugriy** [comment]
thank you for the supersonic review, @vladmos!

**Pull request merged**